test/each.spec.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-env mocha */
+const StringHelper = require('../index')
+const expect = require('chai').expect
+const sinon = require('sinon')
+
+describe('#each', () => {
+  let data = [
+    {a: 1, b: 'John'},
+    {a: 2, b: 'Jane'},
+    {a: 3, b: 'Lisa'}
+  ]
+
+  let table = '<table><thead>' +
+  '<tr><th>ID</th><th>Name</th></tr>' +
```

Comment:
I actually did this to not deal with spacing issues, keep the whole html markup in one single string, but in the test file keep it in different lines. In any case, rearranged the text and improved the test to take into account the spacing. Commit 1bc7ce6